src/catacomb.rs

```diff
@@ -129,6 +133,13 @@ impl<B: Backend + 'static> Catacomb<B> {
         // XDG output protocol.
         xdg::init_xdg_output_manager(&mut display, None);
 
+        // Run user startup script.
+        if let Some(mut home) = dirs::config_dir() {
+            home.push(Path::new("catacomb"));
+            home.push(Path::new(POST_START_SCRIPT));
+            let _ = daemon::spawn("/bin/sh", [home.into_os_string()]);
```

Comment:
Should we check the file exists before executing? Also we don't actually need `/bin/sh`, right? The script should just be an executable anyway, so we can execute it directly.

Reply:
Hm, I was thinking of having it non executable as well. So that's why it's a bin 'sh', but I guess we can assume that it's an executable.